auth-server/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "auth-server",
   "version": "1.0.0",
-  "description": "oauth 2.0 server for governance platform",
+  "description": "oauth 2.0 server for polkassembly",
   "scripts": {
-    "test": "knex migrate:latest && mocha -r ts-node/register test/**/*.spec.ts",
+    "test": "./test.sh",
```

Comment:
what is the use of yarn test

Reply:
what should be the content of test.sh

Reply:
I decided to do that because there would be a bunch of things to do. But you're right, things could be done in the script directly.

Although note that this PR didn't fix all my problems. When running `yarn test` docker will:
> Recreating auth-server_postgres_1 ... done

.. and then close the container.. which effectively messes up with `auth-server_postgres_1` I had been running. I don't get why it doesn't create a "test" one..